src/components/commons/KiwiDropdown.vue

```diff
@@ -86,7 +95,40 @@ export default {
 </script>
 
 <style scoped>
-.select {
+DockLayout.select {
+    height: 40;
+    padding: 0;
+    background-color: var(--input-bg);
+    border-radius: var(--roundness);
+    border-width: 2;
+    border-color: var(--input-border);
+}
```

Comment:
In this class, only `padding: 0;` is needed. The rest comes from `.input` class (which is better for consistency).